eth/stagedsync/stage_polygon_sync.go

```diff
@@ -1355,6 +1355,7 @@ type polygonSyncStageForkChoiceState int
 
 const (
 	forkChoiceCached polygonSyncStageForkChoiceState = iota
+	forkChoiceNoOpConnected
```

Comment:
I wouldn't call this `NoOp` since we are updating the DB, perhaps something like `forkChoiceUpdateHeader`?

Reply:
done